test/unit/eigensolver/test_reduction_to_band.cpp

```diff
@@ -48,10 +50,20 @@ ::testing::Environment* const comm_grids_env =
     ::testing::AddGlobalTestEnvironment(new CommunicatorGrid6RanksEnvironment);
 
 template <class T>
-struct ReductionToBandTestMC : public TestWithCommGrids {};
+struct ReductionToBandTest : public TestWithCommGrids {};
+
+template <class T>
+using ReductionToBandTestMC = ReductionToBandTest<T>;
```

Comment:
Isn't this supposed to be `using ReductionToBandTestMC = ReductionToBandTest<Backend::MC>` ?

Reply:
ReductionToBandTest is a GTest test parameterized over T, which is the type of the value used in test.

For us T is an element of `MatrixElementTypes`, on the other end we are going to have for each `Backend` an indepdendent test (because we also want to turn them off when `DLAF_WITH_CUDA=off`).

The `using` is just for giving a "self-documenting" name to the generic test, together with respecting the DRY principle by re-using exactly the same generic test (very simple in this case) for both Backend::MC and Backend::GPU.

I don't know if I was clear enough 😅